lang/cs/Org.Apache.REEF.Wake/Remote/Proto/WakeRemoteProtosGen.cs

```diff
@@ -29,27 +29,27 @@
 // Generated from: src/main/proto/RemoteProtocol.proto
 namespace Org.Apache.REEF.Wake.Remote.Proto
 {
-  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name=@"WakeMessagePBuf")]
+  [global::System.Serializable, global::ProtoBuf.ProtoContract(Name = @"WakeMessagePBuf")]
```

Comment:
File WakeRemoteProtosGen.cs is generated from proto file, and this PR changes it. I've checked that if it has spacing issue, Wake project doesn't build. I think we shouldn't change any generated files, but rather exclude them